spec/unit/views/index_as_blog_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+include ActiveAdmin
+describe ActiveAdmin::Views::IndexAsBlog do
+  subject { described_class.new }
+
+  describe '#build' do
+    let(:page_presenter) { double('page_presenter', block: nil) }
+    let(:collection) { double('collection') }
+
+    before do
+      expect(subject).to receive('build_posts')
+      expect(subject).to receive('add_class').with('index')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.